project_script/main.py

```diff
@@ -1,108 +1,105 @@
-import os.path
-import numpy as np
 import pandas as pd
 import matplotlib.pyplot as plt
 from scipy import integrate
 import ask
-
-
-def separate_even_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 1]]
-
-
-def separate_odd_columns(data_frame):
-    return data_frame.iloc[:, [i for i in range(len(data_frame.columns)) if i % 2 == 0]]
-
-
-def is_valid_file(file_path):
-    return True if file_path != "" and os.path.exists(file_path) else False
+import datatreatment as dt
+import constantvalues as cv
```

Comment:
I can't remember, if it's a good practice the names all in lowercase, check pls.

Reply:
According to PIP8, to python, it is!